Fix wrong downgrades of libgxps

ManuelHu opened the pull request:
This should stop the extra "upgrade" of `libgxps`, as found in #115, #114 and olders.

hadess commented:
Would be great if you could document why this happens, either through a fixed issue, or flatpak-external-data-checker documentation.

ManuelHu commented:
Sorry @hadess , I was actually wrong (yes it is a bug in flatpak-external-data-checker, but this proposed fix does not fix it...) I have updated this PR to actually solve the issue.

The issue for f-e-d-c had been reported beforehand, and I added an explanation: https://github.com/flathub/flatpak-external-data-checker/issues/338

The problem is apparently, that old gnome releases were considered pre-release with an odd minor version number, a check that libgxps is hitting with their current version.

ManuelHu commented:
implemented in 06992cd
